.github/workflows/all.yml

```diff
@@ -14,43 +14,20 @@ defaults:
     shell: bash
 
 jobs:
-  test-build:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
-      - name: Install Go
-        uses: actions/setup-go@v2
-        with:
-          go-version: 1.17.x
-      - name: Check build
-        run: |
-            go version
-            pwd && ls -l
-
-            export CGO_ENABLED=0
-            
-            go install go.k6.io/xk6/cmd/xk6@master
-
-            GOPRIVATE="go.k6.io/k6" xk6 build \
-              --output ./k6ext \
-              --with github.com/grafana/xk6-output-prometheus-remote="."
-            ./k6ext version
 
   lint:
     runs-on: ubuntu-latest
     env:
-        GOLANG_CI_VERSION: "1.43.0"
-        GO111MODULE: 'on'
+        GOLANG_CI_VERSION: "1.49.0"
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           fetch-depth: 0
       - name: Install Go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v3
         with:
-          go-version: 1.17.x
+          go-version: 1.18.x
       - name: Install golangci-lint
         working-directory: /tmp
         run: go install "github.com/golangci/golangci-lint/cmd/golangci-lint@v$GOLANG_CI_VERSION"
```

Comment:
It's not critical, but if we use the action, like:
```
- name: golangci-lint on Go
        uses: golangci/golangci-lint-action@v3
        with:
          version: ${{ $GOLANG_CI_VERSION }}
```
It benefits us by commenting in PRs, but it's not a big deal